oauth2_provider/forms.py

```diff
@@ -80,7 +80,7 @@ def clean(self):
             except User.DoesNotExist:
                 user = None
 
-        if user is None or not user.is_active:
```

Comment:
Can we take this opportunity to provide a proper flag for email verification? Co-opting `is_active` is just dirty.